index.bs

```diff
@@ -1718,6 +1719,31 @@ session.SubscriptionRequest = {
 The <code>session.SubscriptionRequest</code> type represents a request to
 subscribe to or unsubscribe from a specific set of events.
 
+#### The session.Extension Type #### {#type-session-Extension}
+
+<pre class="cddl remote-cddl local-cddl">
+Extension = text
+</pre>
+
+The <code>Extension</code> type is base64 encoded web-extension archive (xpi/crx/zip),
+an absolute path to the web-extension archive or absolute path to the unpacked web-extension.
+
+<pre class="cddl remote-cddl local-cddl">
+ExtensionDetails = {
+   extension: Extension,
+   ? installTemporary: bool .default true
```

Comment:
I think we might be better off only supporting temporary installs at first. I don't really have an automation use case in mind where it makes sense to permanently install an extension in a specific profile, and we can always add it if there is one.

Reply:
@jgraham would a temporary install be something new defined by the webdriver bidi spec or do you already have a notion of "temporary" extensions in Firefox (I guess they auto-uninstall)?

@oliverdunk do we have something like this within extensions?

Reply:
I think having this set to `false` would be the current behavior - unpacked extensions stay installed between restarts as long as you use the same profile. I can imagine how `true` might work but I'm not sure I'm convinced of the value of that, at least for the MVP.